teams/4/page-2.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="style.css">
+
+</head>
+<body>
+    <h1 class="heady"></h1>
```

Comment:
For which text is this heading? As far as I see, in my opinion, "Billing" should be H1 in this page, but in that case this tag should no be here structurally.